### PR TITLE
Add November 2022 Updates

### PR DESCRIPTION
**Pull request by mikeebowen**
- Added DocumentFormat.OpenXml.Office.SpreadSheetML.Y2022.PivotRichData namespace
- Added DocumentFormat.OpenXml.Office.PowerPoint.Y2019.Main.Command namespace
- Added DocumentFormat.OpenXml.Office.PowerPoint.Y2022.Main.Command namespace
- Added Child RichDataPivotCacheGuid to DocumentFormat.OpenXml.Office2010.Excel.PivotCacheDefinition
- Fixed .NET Core Remote Code Execution Vulnerability (#1222)

### CODE REVIEW

**Comment by twsouthwick** [comment]
@mikeebowen merge changes from main. Make sure to remove the files marked as not in sync:

```
generated/DocumentFormat.OpenXml.Features/Microsoft.Interop.JavaScript.JSImportGenerator/Microsoft.Interop.JavaScript.JSExportGenerator/JSExports.g.cs
```

and similar

**Comment by twsouthwick** [comment]
You'll also want to rerun the build locally to get some updates to the changed generated files

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,12 +8,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- Added DocumentFormat.OpenXml.Office.PowerPoint.Y2019.Main.Command namespace
+- Added DocumentFormat.OpenXml.Office.PowerPoint.Y2022.Main.Command namespace
+- Added Child RichDataPivotCacheGuid to DocumentFormat.OpenXml.Office2010.Excel.PivotCacheDefinition
 
 ### Fixed
 - Removed reflection usage where possible (#1240)
 - Fixed issue where some URIs might be changed when cloning or creating copy (#1234)
 - Fixed issue in FlatOpc generation that would not read the full stream on .NET 6+ (#1232)
 - Fixed issue where restored relationships wouldn't load correctly (#1207)
+- Fixed .NET Core Remote Code Execution Vulnerability (#1222)
```
Comment (twsouthwick) [inline CHANGELOG.md:21]: This is not true - this was in a test dependency, not anything we shipped externally

**Review by twsouthwick — CHANGES_REQUESTED** [review]
Other than the CHANGELOG line we should remove, LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (mikeebowen) [reply CHANGELOG.md]: removed

**Review by twsouthwick — APPROVED** [review]

**Comment by twsouthwick** [comment]
@mikeebowen any thing preventing this from merging at this point?

**Comment by tomjebo** [comment]
@twsouthwick it should be ok to merge. We discussed this morning and I was thinking we wanted review of the PR by external contributors but it's not necessary, they can review a draft release with all the changes for 2.19.

**Comment by twsouthwick** [comment]
You'll want to merge from main and rebuild to get some updated changes to the part generation I've merged in

**Pull request merged**